packages/app/src/runner/useEventManager.ts

```diff
@@ -51,6 +35,8 @@ export function useEventManager () {
     eventManager.on('visit:blank', ({ type }) => {
       getAutIframeModel().visitBlank({ type })
     })
+
+    eventManager.on('expect:origin', addCrossOriginIframe)
```

Comment:
original in [runner/src/iframe/iframes.jsx](https://github.com/cypress-io/cypress/pull/18075/files#diff-bd8f567e7cefe6212c99b7760b613f12ffa518138f1d19aff5f3e5cf0c677bf9R95) now bound here in the `useEventManager` code (seems to fit with other events, like `script:error` and `visit:blank`)